Extending/FileSystemProviders/Azure-Blob-Storage/index-v9.md

```diff
@@ -0,0 +1,116 @@
+
+Setup consists of adding a package to your site, setting the correct configuration, and adding the services and middleware. Before you begin you’ll need to create an Azure Storage Account and a container for your media and ImageSharp cache. In this example we assume your container name is "media". You can, optionally, enable an Azure CDN for this storage container and use it in the appsettings.json below.
+
+See [Microsoft documentation](https://docs.microsoft.com/en-us/azure/storage/blobs/storage-quickstart-blobs-portal) for information on how you set up blob storage container. 
+
+## Installing the package
+
+First things first we need to install the `Umbraco.StorageProviders.AzureBlob` nuget package, there's two approaces to this. You can use your favourite IDE and open up NuGet package manager, and find it there, but you can also use the commandline to install the package.
+
+### Installing through commandline
+
+First you need to navigate to your project folder, that is the folder that contains your `.csproj` file. Now use the dotnet add package command like so:
+
+```
+dotnet add package Umbraco.StorageProviders.AzureBlob
+```
+
+and the correct package will been installed in your project.
+
+
+## Configuring Blob storage
+
+The next step is to configure your blob storage, there's multiple approaches for this, but in this document we're going to do it through `appsettings.json`, for more configuration option see the [readme](https://github.com/umbraco/Umbraco.StorageProviders#umbracostorageproviders) on the github repository.
+
+Open up your `appsettings.json` file, we need to add the connection string and container name under `Umbraco:Storage:AzureBlob:Media`, your Umbraco section of appsettings will look something like this:
+
+```json
+  "Umbraco": {
+    "Storage": {
+      "AzureBlob": {
+        "Media": {
+          "ConnectionString": "DefaultEndpointsProtocol=https;AccountName=<media account name>;AccountKey=<media account key>;EndpointSuffix=core.windows.net",
+          "ContainerName": "media"
+        }
+      }
+    },
+    "CMS": {
+      "Hosting": {
+        "Debug": false
+      },
+      {...}
+    }
+  }
+```
+
+Note that in this case the container name is media. 
+
+Tip: You can get your connection string from Azure under "Access Keys"
+
+## Setting the services and middleware
+
+We're almost there, the last step we need to do is to setup the required services and middleware, this may sound daunting, but thankfully there is extension methods that does all this for us, all we need to do is invoke them in the `ConfigureServices` and `Configure` methods in the `startup.cs` file.
```

Comment:
```suggestion
## Setting the services and middleware

We're almost there, the last step we need to do is to set up the required services and middleware, this may sound daunting, but thankfully there are extension methods that do all this for us, all we need to do is invoke them in the `ConfigureServices` and `Configure` methods in the `startup.cs` file.
```